ext/standard/url.c

```diff
@@ -190,8 +190,8 @@ PHPAPI php_url *php_url_parse_ex(char const *str, int length)
 			memcpy(port_buf, p, (pp - p));
 			port_buf[pp - p] = '\0';
 			port = strtol(port_buf, NULL, 10);
-			if (port > 0 && port <= 65535) {
-				ret->port = (unsigned short) port;
+			if (port > 0 && port <= 4294967295) {
```

Comment:
I think this is an ABI break anyway. It's not just about the struct size but also about the offset of the path, query and fragment that would change here. It would be fine if port was the last but it's not the case so I don't think this should go to 5.6 or 7.0. It should go only to 7.1 if anything.